src/lib/deploy.js

```diff
@@ -316,7 +326,7 @@ async function deploy({ alias, yes }) {
     log('  Using the cached verification key');
   }
 
-  let { fee } = config.networks[alias];
+  let { fee } = config[deployAliasesConfigName][alias];
```

Comment:
I think this would be easier to read if values were accessed at `config.deployAliases` consistently throughout the code. Then lines 45-48 would be updated to check if a `deployAliases` property exists, and if not, assigns the value of `config?.networks` to it.

i.e.
`if (!config.hasOwnProperty('deployAliases')) config.deployAliases = config?.networks;`

Naming is tricky here. We probably benefit from using a consistent pattern to name vars that distinguishes between property name (`deployAliasName`) and the object value (`deployAlias`) too.

Reply:
I agree this approach is more readable. The downside is if a user has an existing `networks` in their `config.json`, an additional `deployAliases` is added to the config.json after going through the `zk config` flow.

Reply:
> The downside is if a user has an existing networks in their config.json, an additional deployAliases is added to the config.json after going through the zk config flow.
But only as a property on the temporary object. If we were writing it back to disk, that'd be a problem, but we don't so I think it's fine.

Reply:
If a user has an existing networks config like this. 

`{
  "version": 1,
  "networks": {
    "networks-berkeley": {
      "url":   "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/networks-berkeley.json",
      "fee": ".1"
    }
  }
}`

They run zk config and add the deploy alias berkeley2. The resulting config.json with a networks and deploy alias is below. 

`{
  "version": 1,
  "networks": {
    "networks-berkeley": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/networks-berkeley.json",
      "fee": ".1"
    },
    "berkeley2": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/berkeley2.json",
      "fee": ".1"
    }
  },
  "deployAliases": {
    "networks-berkeley": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/networks-berkeley.json",
      "fee": ".1"
    },
    "berkeley2": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/berkeley2.json",
      "fee": ".1"
    }
  }
}`

Reply:
Idea: How about when running `zk config`, we check if `networks` property exists and replace with `deployAliases` when writing it to the file, thereby transitioning the config. Then someday in the future (1yr+), we can clean up and remove this code and just assume `deployAliases` exists and let _really, really_ old ones break. Or we could use the version number, but I kinda like the former. 

wdyt?

Reply:
We should take the first approach. That is what I meant when I said this earlier but I didn't communicate it clearly. 

> Another option could be to copy the networks config into a new deployAliases config and delete the network config after.

Reply:
sounds good to me!